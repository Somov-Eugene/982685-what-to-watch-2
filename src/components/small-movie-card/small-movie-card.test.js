import React from "react";
import renderer from "react-test-renderer";
import SmallMovieCard from "./small-movie-card";

describe(`Проверка <SmallMovieCard>`, () => {
  it(`компонент <SmallMovieCard> корректно отрисован`, () => {
    const movie = {name: `Fantastic Beasts`};

    const tree = renderer
      .create(<SmallMovieCard
        movie = {movie}
      />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
