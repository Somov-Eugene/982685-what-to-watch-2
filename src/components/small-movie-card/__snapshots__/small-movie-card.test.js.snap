// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Проверка <SmallMovieCard> компонент <SmallMovieCard> корректно отрисован 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <div
    className="small-movie-card__image"
  >
    <video
      height={175}
      muted={true}
      poster="img/fantastic-beasts-the-crimes-of-grindelwald.jpg"
      src="https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm"
      width={280}
    />
  </div>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="movie-page.html"
      onClick={[MockFunction]}
    >
      Fantastic Beasts: The Crimes of Grindelwald
    </a>
  </h3>
</article>
`;
